models/project-extension/src/builder.spec.ts

```diff
@@ -0,0 +1,111 @@
+        categoryRecommendationSettings: expect.objectContaining({
+          id: expect.any(String),
+          createdAt: expect.any(String),
+          updatedAt: expect.any(String),
+          searchProperty: expect.any(String),
+          attributeName: expect.any(String),
+        }),
+        imageRegex: expect.objectContaining({
+          id: expect.any(String),
+          createdAt: expect.any(String),
+          updatedAt: expect.any(String),
+          small: expect.objectContaining({
+            flag: expect.any(String),
+            replace: expect.any(String),
+            search: expect.any(String),
+          }),
+          thumb: expect.objectContaining({
+            flag: expect.any(String),
+            replace: expect.any(String),
+            search: expect.any(String),
+          }),
+        }),
+        sampleDataImport: expect.objectContaining({
+          completed: expect.any(Boolean),
+          dataset: expect.any(String),
+        }),
+        orderStatesVisibility: expect.arrayContaining([expect.any(String)]),
+      })
+    )
+  );
+
+  it(
+    ...createBuilderSpec<TProjectExtension, TProjectExtensionGraphql>(
+      'graphql',
+      ProjectExtension.random(),
+      expect.objectContaining({
+        __typename: 'ProjectExtension',
+        id: expect.any(String),
+        createdAt: expect.any(String),
+        projectKey: expect.any(String),
+        isRichTextEditorEnabled: expect.any(Boolean),
+        installedApplications: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        installedCustomViews: expect.arrayContaining([
+          expect.objectContaining({
+            id: expect.any(String),
+          }),
+        ]),
+        categoryRecommendationSettings: expect.objectContaining({
+          __typename: 'CategoryRecommendationSettings',
+          id: expect.any(String),
+          createdAt: expect.any(String),
+          updatedAt: expect.any(String),
+          searchProperty: expect.any(String),
+          attributeName: expect.any(String),
+        }),
+        imageRegex: expect.objectContaining({
+          __typename: 'ImageRegex',
+          id: expect.any(String),
+          createdAt: expect.any(String),
+          updatedAt: expect.any(String),
+          small: expect.objectContaining({
+            flag: expect.any(String),
+            replace: expect.any(String),
+            search: expect.any(String),
+          }),
+          thumb: expect.objectContaining({
+            flag: expect.any(String),
+            replace: expect.any(String),
+            search: expect.any(String),
+          }),
+        }),
+        sampleDataImport: expect.objectContaining({
+          __typename: 'SampleDataImportMetadata',
+          completed: expect.any(Boolean),
+          dataset: expect.any(String),
+        }),
+        orderStatesVisibility: expect.arrayContaining([expect.any(String)]),
+      })
```

Comment:
Perhaps for consistency we could also add the 2 missing fields: `updatedAt`, `applications`.

Reply:
updated [0540eed](https://github.com/commercetools/test-data/pull/562/commits/0540eed56dc1eede9d0529dce34832771d5bd58b), thanks.